### PR TITLE
Parsers V2: Refactor KeypointParser

### PR DESCRIPTION
**Pull request by kkeroo**
This PR refactors the Keypoint parser and adds `Keypoints` message as discussed. `Keypoints` msg is a list of `Keypoint` and each `Keypoint` has x,y,z,confidence.

### CODE REVIEW

File: depthai_nodes/ml/parsers/keypoints.py [hunk 1]
```diff
@@ -70,6 +89,14 @@ def setNumKeypoints(self, n_keypoints):
         """
         self.n_keypoints = n_keypoints
```
Comment (aljazkonec1) [inline depthai_nodes/ml/parsers/keypoints.py:90]: setScaleFactor should set scale_factor not n_keypoints, need to add setNumberKeypoints()

**Review by aljazkonec1 — COMMENTED** [review]
OK, just add the setNumberKeypoints() method.

File: depthai_nodes/ml/parsers/keypoints.py (hunk 1 above)
Comment (kkeroo) [reply depthai_nodes/ml/parsers/keypoints.py:90]: It does. When reviewing some lines that are not affected are hidden :)

Comment (aljazkonec1) [reply depthai_nodes/ml/parsers/keypoints.py:90]: My bad, the hidden lines matched up  perfectly.

**Review by jkbmrz — APPROVED** [review]
LGTM!

**Pull request closed without merging**